src/analyzer/cspParser.js

```diff
@@ -24,6 +26,9 @@ export function parseCspMeta(cspList) {
 }
 
 /**
+ * The returned Map has the directive as the key and a Set of sources as the value.
+ * If there are allowed duplicates detected, the first one is kept and the rest are discarded,
+ * and an entry in the final Map is added with the key "_observatory_warnings" and the value.
```

Comment:
Sync the key when finalized, or omit it altogether.

```suggestion
 * and an entry in the final Map is added with the key "_observatory_duplicate_key_warnings" and the value.
```